src/bot.ts

```diff
@@ -726,13 +726,13 @@ export class DiscordBot {
 
         const relatesTo = event.content['m.relates_to'];
 
+        let emoji = relatesTo.key
+        if (emoji.indexOf("👍") >=0  ) {
+            emoji = "👍"
+        } 
```

Comment:
After reading this if statement, I'm assuming that this is related to the skin-tones? If so, you may want to do proper parsing of those, as more than thumbs up is affected. There's also country flags which use a similar concept.